app/main.development.js

```diff
@@ -10,14 +11,26 @@ let debug = false;
 let dcrdPID;
 let dcrwPID;
 
+// Not oing to make incorrect options fatal since running in dev mode has
+// all sorts of things on the cmd line that we don't care about.  If we want
```

Comment:
In dev mode, this is argv:
```
[ '/home/jcv/code/decred/decrediton/node_modules/electron/dist/electron',
  '-r',
  'babel-register',
  '-r',
  'babel-polyfill',
  './app/main.development' ]
```
so electron things that aren't a problem in production mode but things we don't ever directly need to worry about.